clang/lib/Driver/ToolChains/PS4CPU.cpp

```diff
@@ -376,24 +382,32 @@ void toolchains::PS4PS5Base::addClangTargetOptions(
     else
       CC1Args.push_back("-fvisibility-dllexport=protected");
 
+    // For PS4 we override the visibilty of globals definitions without
```

Comment:
Note for reviewer: This comment is new and replaces a comment in our downstream code that wasn't suitable for upstream as it referenced internal tracking numbers.